docs/getting-started.md

```diff
@@ -49,6 +47,20 @@ In this guide, we've documented two different ways to work with OpenSAFELY:
      * you want to try OpenSAFELY via a device other than a desktop or
        laptop computer, e.g. a tablet.
 
+     To use Gitpod, the only requirements are:
+
+     * you have a current version of a modern web browser (e.g. Chrome,
+       Edge, Firefox, Safari);
+     * your network connection does not block access to the Gitpod
+       development environment.
+
+     If your internet connection is not managed by you directly —
+     perhaps you are in the office or connecting via your employer's
+     virtual private network (VPN) — it may be that [corporate firewalls
+     prevent access to
+     Gitpod](https://github.com/opensafely/documentation/discussions/311);
+     please contact the IT staff who manage the network for help.
```

Comment:
I think `the network` is slightly ambigious here - how about the same phrase used earlier in the paragraph (i.e. `your internet connection` )? Possibly poor style in conventional writing, but I think better in this context?